docs/spring-boot/core-starter.md

```diff
@@ -0,0 +1,257 @@
+
+## Subscribe to Real Time Events
+The Core Starter uses [Spring Events](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/context/ApplicationEventPublisher.html) 
+to deliver Real Time Events. 
+
+You can subscribe to any Real Time Event from anywhere in your application by creating a handler method that has to 
+respect two conditions: 
+- be annotated with [@EventListener](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/context/event/EventListener.html) 
+- have `com.symphony.bdk.spring.events.RealTimeEvent<T>` parameter
+
+Here's the list of Real Time Events you can subscribe:
+```java
+@Component
+public class RealTimeEvents {
+
+  @EventListener
+  public void onMessageSent(RealTimeEvent<V4MessageSent> event) {}
+
+  @EventListener
+  public void onSharedPost(RealTimeEvent<V4SharedPost> event) {}
+
+  @EventListener
+  public void onInstantMessageCreated(RealTimeEvent<V4InstantMessageCreated> event) {}
+
+  @EventListener
+  public void onRoomCreated(RealTimeEvent<V4RoomCreated> event) {}
+
+  @EventListener
+  public void onRoomUpdated(RealTimeEvent<V4RoomUpdated> event) {}
+
+  @EventListener
+  public void onRoomDeactivated(RealTimeEvent<V4RoomDeactivated> event) {}
+
+  @EventListener
+  public void onRoomReactivated(RealTimeEvent<V4RoomReactivated> event) {}
+
+  @EventListener
+  public void onUserRequestedToJoinRoom(RealTimeEvent<V4UserRequestedToJoinRoom> event) {}
+
+  @EventListener
+  public void onUserJoinedRoom(RealTimeEvent<V4UserJoinedRoom> event) {}
+
+  @EventListener
+  public void onUserLeftRoom(RealTimeEvent<V4UserLeftRoom> event) {}
+
+  @EventListener
+  public void onRoomMemberPromotedToOwner(RealTimeEvent<V4RoomMemberPromotedToOwner> event) {}
+
+  @EventListener
+  public void onRoomMemberDemotedFromOwner(RealTimeEvent<V4RoomMemberDemotedFromOwner> event) {}
+
+  @EventListener
+  public void onConnectionRequested(RealTimeEvent<V4ConnectionRequested> event) {}
+
+  @EventListener
+  public void onConnectionAccepted(RealTimeEvent<V4ConnectionAccepted> event) {}
+
+  @EventListener
+  public void onMessageSuppressed(RealTimeEvent<V4MessageSuppressed> event) {}
+
+  @EventListener
+  public void onSymphonyElementsAction(RealTimeEvent<V4SymphonyElementsAction> event) {}
+}
+```
+
+## Inject Services
+The Core Starter injects services within the Spring application context:
+```java
+@Service
+public class CoreServices {
+    
+    @Autowired
+    private MessageService messageService;
+    
+    @Autowired
+    private StreamService streamService;
+    
+    @Autowired
+    private UserService userService;
+    
+    @Autowired
+    private DatafeedService datafeedService;
+    
+    @Autowired
+    private SessionService sessionService;
+    
+    @Autowired
+    private ActivityRegistry activityRegistry;
+}
+```
+> Please note that field injection is not recommended, you should consider using 
```

Comment:
Then perhaps we should not show an example using field injection :) (but it is shorter...) 
So maybe we could just list the available services?